src/displayapp/screens/SystemInfo.cpp

```diff
@@ -43,10 +43,9 @@ bool SystemInfo::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
 }
 
 std::unique_ptr<Screen> SystemInfo::CreateScreen1() {
-  auto batteryPercentF = batteryController.PercentRemaining();
-  uint16_t batteryPercent = 0;
-  if(batteryPercentF > 100.0f) batteryPercent = 100;
-  else if(batteryPercentF < 0.0f) batteryPercent = 0;
+  auto batteryPercent = batteryController.PercentRemaining();
+  if(batteryPercent > 100.0f) batteryPercent = 100;
+  else if(batteryPercent < 0.0f) batteryPercent = 0;
```

Comment:
We can remove the boundary check completely, as it is already checked by `Battery::Update()`

Reply:
Good catch... looks like

https://github.com/JF002/Pinetime/blob/f0e1f98823e41bfc2d9743fa8de70c882f26f93b/src/components/battery/BatteryController.cpp#L39-L40

will prevent it going out of range, and can thus be used directly. This makes it so only the conversion from float to int is needed and reducing this down to just `int8_t batteryPercent = (int)batteryController.PercentRemaining();` ??